.eslintrc

```diff
@@ -25,7 +25,7 @@
         "semi": ["error", "never"],
         "quotes": ["error", "single"],
         "vue/multi-word-component-names": "off",
-        "max-len": ["error", { "code": 999 }]
+        "max-len": ["error", { "code": 180 }]
```

Comment:
Thoughts on max-len 120 and quotes double?

Reply:
I like 180 because:
 - I don't love line wrapping when it could easily be read on one line
 - typescript increases verbosity and could cause line wrapping more often making readability worse, imho
 - we use a decent amount of highly descriptive (verbose) types/interfaces and method names that could increase wrapping (this is purely a hot take; I know 80-120 was commonplace in lots of javascript projects, but perhaps that became commonplace pre-typescript popularity?) 🤷‍♂️ 
 - on default zoom, with no other windows open, the VSCode IDE shows ~180 characters (columns) on the 13" macbook we all (most of us?) are using
 - the monitor I use (anyone else using external monitor?) shows 200+ characters (columns) with one enhanced zoom level

That said, totally open to 120 (or 150?) if you have strong preference.

I prefer single quotes because it looks cleaner to me (and we're not messing with escape characters too much outside of some html copy), but no strong feelings. If you feel strongly about double, let's do it.